Gems/AtomTressFX/Assets/Shaders/HairRenderingResolvePPLL.shader

```diff
@@ -0,0 +1,55 @@
+{ 
+    "Source" : "HairRenderingResolvePPLL.azsl",
+	
+    "CompilerHints" : 
+    { 
+        "DxcDisableOptimizations" : false,
+        "DxcGenerateDebugInfo" : false
+    },
+
+    "DepthStencilState" : 
+    {
+        "Depth" : 
+        { 
+            "Enable" : true,    // The resolve will write the closest hair depth
+            // Pixels that don't belong to the hair will be discarded, otherwise if a fragment
+            // exist in the list, it already passed the depth test
+            "CompareFunc" : "Always"    
+        },
+        "Stencil" :
+        {
+            "Enable" : false
+        }
+    },
+
+    "BlendState" : 
+    {
+        "Enable" : false,
+//        "Enable" : true,
```

Comment:
For referecne as this is the mode required by TressFX.  In our implementation I ended up pre-multiplying and alpha blended with the back buffer to avoid the TressFX silhouette.

Reply:
removed and commented